app/javascript/elm/src/Main.elm

```diff
@@ -226,6 +232,9 @@ type Msg
     | HighlightSessionMarker (Maybe Location)
     | GraphRangeSelected (List Float)
     | UpdateIsShowingTimeRangeFilter Bool
+    | SaveScrollPosition Float
+    | SetScrollPosition
```

Comment:
So one of those two branches above sets and saves the scroll position on the model, and the other on the sessions list viewport? Have you considered making `SetScrollPosition` name a bit more specific and explicit about that? Just my 2 cents, it looks good either way!

Reply:
Thanks, that's a good point. I'll try to find better names :)